nextflow.config

```diff
@@ -20,6 +21,14 @@ params {
     max_multiqc_email_size     = '25.MB'
     multiqc_methods_description = null
 
+// Fastp parameters
+    fastp_save_trimmed_fail                 = false
+    fastp_qualified_quality                 = 15
+    fastp_cut_mean_quality                  = 15
+    adapterremoval_minquality               = 2
+    keep_phix                               = false
+    adapterremoval_trim_quality_stretch     = false
```

Comment:
wait why is this false? what does that do?

Reply:
I need to check this